server/src/db/repositories/reportRepository.js

```diff
@@ -274,21 +289,170 @@ function getReportDetailsPipeline(id) {
                     $filter: {
                       input: '$testCases.gherkinDocument.feature.tags.name',
                       cond: {
-                        $regexMatch: { input: '$$tags', regex: /^@endpoint=/ },
+                        $regexMatch: {
+                          input: '$$tags',
+                          regex: /^@endpoint=/,
+                        },
                       },
                       as: 'tags',
                     },
                   },
                 },
+                // cut endpoint ( remove 10 characters from the beginning and take the rest )
                 10,
                 -1,
               ],
             },
             passed: '$testCases.passed',
+            details: [
+              {
+                scenario: '$testCases.name',
+                steps: '$testCases.steps',
+              },
+            ],
           },
         },
       },
     },
+    {
+      $unwind: {
+        path: '$data',
+      },
+    },
+    {
+      $unwind: {
+        path: '$data.details',
+      },
+    },
+    {
+      $unwind: {
+        path: '$data.details.steps',
+      },
+    },
+    {
+      $project: {
+        compatibilities: 1,
```

Comment:
what do these '1' values mean?

Reply:
maybe it's because I'm not this familiar with this syntax, but it's hard for me to read - I think a few comments here and there wouldn't hurt, but I wouldn't hold the PR because of that